pipettes/firmware/motor_timer_hardware.c

```diff
@@ -34,7 +34,7 @@ void MX_TIM6_Init(void) {
     TIM_MasterConfigTypeDef sMasterConfig = {0};
 
     htim6.Instance = TIM6;
-    htim6.Init.Prescaler = 499;
+    htim6.Init.Prescaler = 849;
```

Comment:
Which timer is this? Is it undoing https://github.com/Opentrons/ot3-firmware/pull/464 ?

Reply:
I think accidental unresolved merge.